packages/terra-date-picker/package.json

```diff
@@ -48,7 +48,7 @@
     "compile": "babel --root-mode upward src --out-dir lib --copy-files",
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
-    "lint:scss": "stylelint src/**/*.scss --ignore-path .stylelintignore",
```

Comment:
I am not sure but i presume that we can safely delete the ".stylelintignore" file, since the only purpose of this was to ignore lint checks for the "react-datepicker" directory.
@nramamurth, @supreethmr